README.md

```diff
@@ -5,118 +5,29 @@ Specifically, this enables the development and testing of payload applications:
 
 It is recommended to first read through the [SatOS Payload SDK Guide](./docs/Antaris_SatOS_Payload_SDK_Guide.pdf) to familiarize yourself with the concepts and expected operations of payloads within the SatOS platform.
 
-## Quickstart: SDK Dependencies
+## Local Setup
 
-Before you continue, the following depdencies must be available on your local machine:
+Before you continue, it is assumed that:
+1. You have Docker installed on your local machine
+2. You have access to the Antaris Cloud Platform.
 
-- docker (tested with docker version 20.10.12)
-- make (tested with version GNU Make 4.2.1)
+If you do not have access to the cloud platform, you can still get started with the SDK but you will be limited in what can be accomplished.
 
-The SDK has been tested on both Linux- and OSX-based machines.
+## Quickstart
 
-## Quickstart: Environment Setup
+Visit the [Python example application](./examples/app-python/) to quickly get a working example up and running locally.
 
-Before application development can begin, you must first create a build environment.
-The SDK utilizes Docker to provide a reusable, portable build environment via containerization.
+## Developing Payload Applications
 
-### Launch Build Environment
+The development processes available here utilize Docker containers.
+It is expected that developers start from one of the supported base images:
 
-Use `make build_env` to instantiate a build environment container and open a new shell within it.
+* Python applications: `quay.io/antaris-inc/satos-payload-app-python:stable`
+* C++ application: quay.io/antaris-inc/satos-payload-app-cpp:stable
 
-NOTE: This step also builds the requisite Docker image which requires internet access to install some system packages.
+A developer does not even need to close this repository in order to build an application, however there are examples
```

Comment:
Spelling mistake = "even need to **close** this repository" .. "even need to **clone** this repository"

Reply:
Yes, good catch!